src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/System.Text.RegularExpressions.Tests.csproj

```diff
@@ -43,6 +44,9 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
     <Compile Include="CustomDerivedRegexScenarioTest.cs" />
+    <Compile Include="CSharpCodeFixVerifier`2.cs"/>
+    <Compile Include="UpgradeToRegexGeneratorAnalyzerTests.cs" />
+    <Compile Include="$(CommonTestPath)SourceGenerators\LiveReferencePack.cs" Link="Common\SourceGenerators\LiveReferencePack.cs" />
```

Comment:
Why are these needed?  Why isn't this just like the generator, referencing the public types exposed from the System.Text.RegularExpressions.Generator.csproj being referenced?

Reply:
Not necessarily needed, this is just a different approach from what the source generator tests do. This is the pattern that is followed by the LibraryGenerator analyzer tests, which is just a more centralized way of how to reference the whole live-built ref pack form the tests. That said, I'm fine to remove this complexity and use the source generator tests approach instead, I'll do this in a bit.